lib/report_plugins/treatments.js

```diff
@@ -268,13 +284,14 @@ treatments.report = function report_treatments(datastorage, sorteddaystoshow, op
     .append($('<th>').css('width','50px').attr('align','left').append(translate('Duration')))
     .append($('<th>').css('width','50px').attr('align','left').append(translate('Percent')))
     .append($('<th>').css('width','50px').attr('align','left').append(translate('Basal value')))
+    .append($('<th>').css('width','50px').attr('align','left').append(translate('Profile')))
     .append($('<th>').css('width','150px').attr('align','left').append(translate('Entered By')))
     .append($('<th>').css('width','300px').attr('align','left').append(translate('Notes')))
   );
   
   sorteddaystoshow.forEach(function (day) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1240173027/issues/source?bid=2452379&fileBranchId=2935247#l292)